packages/react-server-dom-webpack/package.json

```diff
@@ -49,6 +49,7 @@
   },
   "peerDependencies": {
     "react": "^17.0.0",
+    "react-dom": "^17.0.0",
```

Comment:
This was not possible to declare before because this also imports implementation details from `react-dom`.